client/scss/components/_tabs.scss

```
@@ -111,53 +9,71 @@
   }
 }
 
-@include media-breakpoint-up(sm) {
-  .tab-nav {
-    // For cases where tab-nav should merge with header
-    &.merged {
-      background-color: $color-header-bg;
-    }
-
-    li {
-      width: auto;
-      padding: 0;
-    }
-
-    a {
-      padding-inline-start: $mobile-nice-padding;
-      padding-inline-end: $mobile-nice-padding;
-    }
-
-    li.settings a {
-      padding-inline-start: 2em;
-      padding-inline-end: 2em;
+.w-tabs {
+  &__wrapper {
+    @apply w-mb-10;
+  }
+
+  &__list {
+    @apply w-flex w-space-x-6 w-border-b w-border-grey-100 w-w-fit w-px-[20px] sm:w-px-[40px] md:w-px-[80px];
+  }
+
+  &__tab {
+    @apply w-label-3
+    w-outline-offset-inside
+    w-inline-flex
+    w-text-grey-400
+    hover:w-text-primary
+    w-whitespace-nowrap
+    w-py-4
+    w-font-medium
+    w-relative
+    after:w-block
+    after:w-w-0
+    after:w-h-[2px]
+    after:w-bg-primary
+    after:w-absolute
+    after:w-left-0
+    after:-w-bottom-px
+    after:w-transition-all
+    motion-reduce:after:w-transition-none
+    hover:after:w-w-full;
+
+    &[aria-selected='true'] {
+      @apply after:w-w-full w-text-primary;
     }
   }
 
-  .modal-content .tab-nav li {
-    padding: 0;
-    min-width: 0;
-
-    &:first-of-type {
-      padding-inline-start: $desktop-nice-padding;
+  &__errors {
+    @apply w-hidden
+    w-box-border
+    w-w-[16px]
+    w-h-[16px]
+    w-text-[10px]
```

Needs to be set in rem, and 10px is definitely too small to be readable.

```suggestion
    w-text-[0.75rem]
```